delivery_boy.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "ruby-kafka", "~> 1.0"
   spec.add_runtime_dependency "king_konf", "~> 0.3"
 
-  spec.add_development_dependency "bundler", "~> 1.15"
+  spec.add_development_dependency "bundler", "~> 2.1"
```

Comment:
This change causes the build to fail. Can you remove it or figure out how to make it work?